src/pages/Defi/hooks/useFoxyBalances.tsx

```diff
@@ -1,65 +0,0 @@
-import { ethChainId, foxAssetId, fromAccountId, toAccountId } from '@shapeshiftoss/caip'
-import { useMemo } from 'react'
-import { useWallet } from 'hooks/useWallet/useWallet'
-import { useWalletSupportsChain } from 'hooks/useWalletSupportsChain/useWalletSupportsChain'
-import { isValidAccountNumber } from 'lib/utils'
-import { useGetFoxyAprQuery, useGetFoxyBalancesQuery } from 'state/apis/foxy/foxyApi'
-import {
-  selectPortfolioAccountIdsByAssetId,
-  selectPortfolioAccountsGroupedByNumberByChainId,
-} from 'state/slices/portfolioSlice/selectors'
-import { useAppSelector } from 'state/store'
-
-export function useFoxyBalances({ accountNumber }: { accountNumber?: number } = {}) {
-  const {
-    state: { wallet },
-  } = useWallet()
-
-  const accountsByNumber = useAppSelector(state =>
-    selectPortfolioAccountsGroupedByNumberByChainId(state, { chainId: ethChainId }),
-  )
-
-  const userAddress = useMemo(() => {
```

Comment:
User address / account number hacks are also a non-problem now.
We now use an `AccountId` as a source of truth, following the abstraction.
What this previously solved (calling investor-foxy with the right input) is now handled at resolver-level using `fromAccountId(accountId).account`, and deriving bip44Params from the account.